src/test/kotlin/com/amazon/ionelement/IonElementLoaderTests.kt

```diff
@@ -57,15 +64,29 @@ class IonElementLoaderTests {
 
             IonElementLoaderTestCase("1", ionInt(1)),
 
-            IonElementLoaderTestCase("existence::42", ionInt(1).withAnnotations("existence")),
```

Comment:
🗺️ A couple of test cases were wrong, but otherwise these changes are just adding test cases for every Ion type.